packages/odf/components/mcg/CreateObjectBucketClaim.tsx

```diff
@@ -367,10 +367,12 @@ export const CreateOBCForm: React.FC<CreateOBCFormProps> = (props) => {
 };
 
 type CreateOBCProps = {
+  formClassName?: string;
   showNamespaceSelector?: boolean;
 };
 
 export const CreateOBC: React.FC<CreateOBCProps> = ({
+  formClassName = '',
```

Comment:
just an optional comment, why can't we just pass `className` itself ?? that way we don't have to be aware of a custom prop name `formClassName`.
```suggestion
  className = '',
```